artemis/reporting/modules/dangling_dns_detector/translations/pl_PL/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,13 @@
+#: artemis/reporting/modules/dangling_dns_detector/template_dangling_dns_record.jinja2:2
+msgid "We identified a dangling dns records for the following subdomains:"
+msgstr "Wykryto ustawienia dangling dns rekordu dla domen:"
```

Comment:
I'd write it as simple as possible, preferably without the "dangling dns" phrase - the technical expertise of the recipients of our e-mails varies a lot.